### PR TITLE
Glossary Improvements

### PR DESCRIPTION
**Pull request by jGleitz**
* Defined Beagle and referenced it everywhere.
 * Added SoMoX’ acronym’s meaning.
 * Added `\begin{quote}` and `\end{quote}` wherever applicable in the glossary.
 * Adds a new package to the required packages on Ubuntu
 * adds `-interaction=nonstopmode` to `pdflatex` so it does not expect input for some errors.

[PDF](http://jgleitz.github.io/Beagle/branches/definition-of-Beagle/Requirements%20Specification.pdf)

### CODE REVIEW

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx [hunk 1]
```diff
@@ -1588,6 +1606,57 @@ cite{PCMtechreport}.
...
+	description={
+\end_layout
+
+\begin_layout Plain Layout
+
+		``
+\backslash
+textbf{BE}haviour 
+\backslash
+textbf{An}alysis using
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: the n should not be in bracktes.

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: fixed, thanks!

**Comment by RomanLangrehr** [comment]
[PDF](http://jgleitz.github.io/Beagle/branches/definition-of-Beagle/Requirements%20Specification.pdf)

**Comment by jGleitz** [comment]
Travis is having hiccups again, hang on!

**Comment by jGleitz** [comment]
The first pdflatex run always fails on Travis. I’ll have to examine that. It doen’t happen on my machine :'(

**Comment by jGleitz** [comment]
Fixed.

**Comment by acBerger** [comment]
lgtm

**Comment by RomanLangrehr** [comment]
Lgtm

**Comment by flyfl** [comment]
Nooooo. That has actually destroyed my new created repo again. Please do not use ":" anymore :(

**Comment by jGleitz** [comment]
Oh no, my bad! I’m so sorry! I obviously made a mistake when merging.

**Pull request merged**